clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -91,6 +91,12 @@ Improvements to clang-tidy
 New checks
 ^^^^^^^^^^
 
+- New :doc:`performance-redundant-lookup
+  <clang-tidy/checks/performance/redundant-lookup>` check.
+
+  This check warns about potential redundant container lookup operations within
```

Comment:
`This check` should be omitted.

Reply:
Dropped in c6a681c955fef8b933374a05a157f48ba4ead360